src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -591,10 +591,11 @@ private void createAndSetStatistics()
 
         findPaneOfTypeByID(HAPPINESS_LABEL, Label.class).setLabelText(roundedHappiness);
         final int citizensSize = townHall.getColony().getCitizenCountLimit();
+        final AdditionModifierResearchEffect citizenCapBoost = this.building.getColony().getResearchManager().getResearchEffects().getEffect(CAP, AdditionModifierResearchEffect.class);
+        final int citizensCap = (int)(Math.min(MineColonies.getConfig().getServer().maxCitizenPerColony.get(), citizenCapBoost != null ? 25 + citizenCapBoost.getEffect() : 25));
 
-        final String numberOfCitizens =
-          LanguageHandler.format(COM_MINECOLONIES_COREMOD_GUI_TOWNHALL_POPULATION_TOTALCITIZENS, citizensSize, townHall.getColony().getCitizenCountLimit());
-        findPaneOfTypeByID(TOTAL_CITIZENS_LABEL, Label.class).setLabelText(numberOfCitizens);
+        findPaneOfTypeByID(TOTAL_CITIZENS_LABEL, Label.class).setLabelText
+                     (LanguageHandler.format(COM_MINECOLONIES_COREMOD_GUI_TOWNHALL_POPULATION_TOTALCITIZENS, citizensSize, townHall.getColony().getCitizenCountLimit(), citizensCap));
```

Comment:
While you're on this, there is a small bug here that was reported, citizensSize is also the count limit, instead of the total number of existing citizens